analysis_options.yaml

```diff
@@ -12,8 +11,8 @@ analyzer:
     # The noise of deprecations isn't worth the effort.
     deprecated_member_use: ignore
   exclude:
-    - .dart_tool/**
-    - build/**
+    - "*/.dart_tool/**"
+    - "*/build/**"
```

Comment:
Did this not work in a specific scenario? Could you double check this still works in both IntelliJ and VS Code? See the last change I made here for context: https://github.com/dart-lang/angular/commit/d1fc081ab540e5d441a107874d5efe29657c17f2.